### ISSUE TITLE
Production

### ISSUE THREAD

**Issue by brochero** [BlancoFS/mkShapesRDF#2]
- [x] Set a stable version of the code to generate a production tag. ~`v0.0.1`~
 -  `v0.0.1` deprecated due to a bug in the DR(lep,jet). Check [commit](https://github.com/BlancoFS/mkShapesRDF/commit/b66019be2c732b6933585aca6447ec1603dc2af3)
- [x] Define samples to be processed: _Link to samples [Google Sheet](https://docs.google.com/spreadsheets/d/1Q3jqmtKuZkWExu3aOShRQtqvtZuZcyiIZmnO-qCsu_I/edit#gid=0)_

**Comment by jmvizan** [comment]
I have tested the instructions provided in #1 and been able to run successfully over the WZ sample. Attached there is a plot of the Muon_pt branch for an output rootfile to show that the output looks okay at first glance (also out.txt shows correctly the summary efficiency tables)
![WZ_Muon_pt](https://github.com/BlancoFS/mkShapesRDF/assets/4331115/6539106b-1cec-4bf5-8494-135c662f8f2b)

**Comment by BlancoFS** [comment]
Thank you @jmvizan for this test, this is great! A further step that would be really good is to see if the Lepton_pt[0]*(Lepton_pdgId[0]==13) distribution is the same as the one you show here. We do not touch the Muon_pt branch but we do for the merged Lepton_pt

**Comment by jmvizan** [comment]
Ty for the suggestion @BlancoFS . I have tried to make a check along those lines:
a) ```Muon_pt[0]*(Muon_pt[0]>Electron_pt[0])```
b) ```Lepton_pt[0]*(Muon_pt[0]>Electron_pt[0])```
c) ```Lepton_pt[0]*(Muon_pt[0]>Electron_pt[0])*(abs(Lepton_pdgId[0]==13))```

a) and b) are very similar but not identical (in particular for low pt). However, c) contains around half of the entries of the other 2 (see attached plot).

I guess there is some interplay between the Id and the pT to form the Lepton collection that explains that the different distributions differ. I don't know well enough the code to be sure of what is the proper easy check to be sure that everything works as expected
![h_MuonPt_validation](https://github.com/BlancoFS/mkShapesRDF/assets/4331115/77da613b-b851-44ea-9c02-d96d687bbd00)

**Comment by BlancoFS** [comment]
Hi again @jmvizan, I think the expression in c) is wrong. The parethesis should be in this way, right?
`Lepton_pt[0]*(Muon_pt[0]>Electron_pt[0])*(abs(Lepton_pdgId[0])==13)`

**Comment by BlancoFS** [comment]
With respect to b) and a). There is a cleaning of lepton inside jets when the Muon and Electron collections are merged, so that's why there may be small differences between the two distributions. I would say this is expected

**Comment by jmvizan** [comment]
You are right about the mistake with c) @BlancoFS . After redoing it a) and c) are really similar (although not identical)

![h_LeptonPt_validation](https://github.com/BlancoFS/mkShapesRDF/assets/4331115/75d00e7b-2339-488c-9559-89e79ca14464)

I think indeed the cleaning of lepton can explain these small differences most likely
